src/Committee/CommitteeMandateCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Committee;
+
+use App\Entity\Adherent;
+use App\Entity\AdherentMandate\CommitteeAdherentMandate;
+use App\Entity\Committee;
+use App\Validator\AdherentForCommitteeMandateReplacement as AssertAdherentValid;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @AssertAdherentValid(errorPath="adherent")
+ */
+class CommitteeMandateCommand
+{
+    /**
+     * @var Adherent|null
+     *
+     * @Assert\NotNull(message="adherent_mandate.adherent.empty")
+     */
+    protected $adherent;
+
+    /**
+     * @var string|null
+     */
+    private $gender;
+
+    /**
+     * @var \DateTime|null
+     */
+    protected $beginAt;
+
+    /**
+     * @var Committee
+     */
+    protected $committee;
+
+    /**
+     * @var string|null
+     */
+    protected $quality;
+
+    public $provisional;
+
+    public static function createFromCommitteeMandate(CommitteeAdherentMandate $mandate): self
+    {
+        $dto = new self();
+        $dto->gender = $mandate->getGender();
+        $dto->committee = $mandate->getCommittee();
+        $dto->quality = $mandate->getQuality();
+        $dto->provisional = $mandate->isSupervisor() ? true : $mandate->isProvisional();
```

Comment:
why not just: 
`$dto->provisional = $mandate->isSupervisor()`
or 
`$dto->provisional = (bool) $mandate->isSupervisor()`